src/components/ui/TextArea/TextArea.tsx

```diff
@@ -135,7 +135,7 @@ const TextArea: React.FunctionComponent<TextAreaProps> = (props) => {
         return <Input.TextArea
             key={key}
             defaultValue={defaultValue}
-            autosize={{
+            autoSize={{
```

Comment:
I think we should keep it `autosize` for now, there is a definite possibility that this will break.

Last 3.*.* version (3.26.13) did indeed deprecated `autosize`, but support both form:
https://github.com/ant-design/ant-design/blob/1c302163afcf2cb97fcb153df9be807fb5200515/components/input/TextArea.tsx#L13

At the same time, previous 3.*.* versions (e.g. 3.18.2) supports only `autosize`:
https://github.com/ant-design/ant-design/blob/6e7c5a95d20cc823417542df3b70272bf708c54c/components/input/TextArea.tsx#L8

As Tesler UI do not have its own instance of antd, client application antd is used, and if they fail to migrate (due to this being minor update, or their own fork) then this property will not work on their side